### ISSUE TITLE
Display media text messages

### ISSUE THREAD

**Issue by mdrxy** [WBOR-91-1-FM/wbor-studio-dashboard#35]

**Comment by CaspianA1** [comment]
This would be interesting

A design issue would be how to display the media

I'm afraid there might not be a good way to do it with the current messaging layout, since the messages displayed will need to take up variable amounts of space, width and height-wise. The current setup doesn't have enough space for images, and allocating bigger message bubbles would just result in less overall space

There's also the question of moderation, and if someone starts sending NSFW images over the dashboard - something like that is bound to happen eventually

Many things to consider before implementing this...

**Comment by mdrxy** [comment]
Something to consider to reduce the complexity of handling variable size media is actually to *not* show the media in-line but instead to indicate (as text) that there are media items attached to a message. We'd need to add some kind of user I/O (perhaps an additional Streamdeck) that includes a button to view media, which then pops up the media (taking up a sizable portion of the screen) for 5-10 seconds for viewing before returning to the default state.

Moderation/etc. will be handled outside of the scope of this repo.

**Comment by CaspianA1** [comment]
Yeah something else for showing images would be ideal

I have a little demo up and running for showing sent images - just for the first attachment sent per message

![image](https://github.com/user-attachments/assets/6083bd99-a1fd-4065-887e-ecaa1f8c648d)

Looks crazy small
So external device would be nice

**Comment by mdrxy** [comment]
Realistically, I don't see this getting done nor do I think it is ultimately necessary. To prevent media messages, I'll have our Twilio auto-respond something along the lines of "media messages aren't supported and will not be delivered to the DJ".

If some future use-case comes up and future maintainers want media display, they can reopen this.

**Comment by CaspianA1** [comment]
Yeah - media just doesn't display well with this layout at all, and it's quite a large endeavor to find a good way to do it here

On the dashboard end, it already displays a message that media messages aren't supported - and when sending an image, it already autosends back some text saying that media messages aren't supported - so this shouldn't be necessary to implement upstream IMO
